src/main/resources/assets/tconstruct/book/fantastic_foundry/index.json

```diff
@@ -35,6 +35,20 @@
       }
     }
   },
+  {
+    "name": "plate",
```

Comment:
If you name the section `armor`, you get an index automatically as that is the name I registered it under. Same for Materials and You